components/button-bar/button-bar.js

```diff
@@ -29,16 +37,15 @@ type PropsType = {
 
 export const ButtonBar = ({ buttonConfigs }: PropsType): React$Element<any> => (
     <View style={ styles.buttonBarHeader }>
-        <View styleName="horizontal flexible">
+        <View style={ styles.buttons }>
```

Comment:
It'd probably be a good future MR task to make a specific class to handle the original 'horizontal flexible' .